src/Smidge.Core/Options/BundleEnvironmentOptions.cs

```diff
@@ -20,14 +20,21 @@ public static BundleEnvironmentOptionsBuilder Create()
             return new BundleEnvironmentOptionsBuilder(options);
         }
 
+
+
+        private readonly IDictionary<string, BundleOptions> _profileOptions;
+
+
         /// <summary>
         /// Constructor, sets default options
         /// </summary>
         public BundleEnvironmentOptions()
         {
+            _profileOptions = new ConcurrentDictionary<string, BundleOptions>();
```

Comment:
Is there a reason for the ConcurrentDictionary if it isn't being exposed via the _profileOptions? This means that the concurrency methods of ConcurrentDictionary are not used, especially within the indexer method. I would assume we'd use GetOrAdd (etc...) instead of double operation of checking existence and then writing.